src/components/TodoListItem/TodoListItem.module.css

```diff
@@ -0,0 +1,34 @@
+.TodoItem {
```

Comment:
PascalCase isn't generally used on css selectors, typically kabob-case is used like `.todo-item`